source/ddbus/thin.d

```diff
@@ -405,7 +405,7 @@ struct DBusAny {
       TypeMismatchException if the DBus type of the current value of the
       DBusAny object is not the same as the DBus type used to represent T.
   +/
-  T get(T)() @property const
+  T get(T)() @property const 
```

Comment:
Space doesn't need to be there, but I guess that's dfmt messing around.

Reply:
oh lol sorry my vscode didn't even show that to me in the diff...